Add switch type primitives (#5390)
OpenRailAssociation/osrd: python/osrd_schemas/README.md

This python library holds schemas definition used by both API and the RailJson Generator.

## Node Types

> From now on, switch_types no longer exist in the db and are added as primitives in the code of services that need them:

Reviewer: I find it strange to formulate it as if it's a change log.

```suggestion
Some node types (named primitives) are defined in the code of services:
```

- In core we find them in `RJSSwitchType.java`.
- In editoast they are found in `src/schema/switch_type.rs`.

> - In the fuzzer, we find them in `fuzzer.py`.

Reviewer 2: The fuzzer only enumerates the switch types to regroup them in a dict. If we add a list of all switch primitives inside `osrd_schemas/switch_types.py` we wouldn't need to enumerate them in other places, which would make it unnecessary to update `fuzzer.py` on a primitive change, it could be removed from this list
